homeassistant/components/onewire/sensor.py

```diff
@@ -68,10 +181,14 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
                         "temperature",
                     )
                 )
+
+    # We have an owfs mounted
     else:
         for family_file_path in glob(os.path.join(base_dir, "*", "family")):
             with open(family_file_path, "r") as family_file:
                 family = family_file.read()
+            if "EF" in family:
+                next
```

Comment:
I'm guessing we want to use `continue`. `next` is not a flow control statement, it's a builtin function.